source/_docs/configuration/templating.markdown

```diff
@@ -119,6 +119,14 @@ Other state examples:
   Paulus is at {{ states('device_tracker.paulus') }}.
 {% endif %}
 
+#check state and compute remaining time before a train departure time (in minutes)
+{% if (states('sensor.train_departure_time') != "unknown" and states('sensor.train_departure_time') != "unavailable") %}
+{{ ( (as_timestamp(states('sensor.train_departure_time')) - as_timestamp(now()))/60.0 ) | round (2) }}
+{% else %}
+unknown
+{% endif %}
```

Comment:
This looks like an extremely overcomplicated example that isn't really helpful in this location IMHO.

What is it you are trying to explain with this example?

Reply:
You're right ...
I just wanted to show an example with a test with unknown/unavailable state. Is the following example better ?
```
#check state and make a processing
{% if (states('sensor.train_departure_time') != "unknown" and states('sensor.train_departure_time') != "unavailable") %}
{{ ... }}
{% else %}
unknown
{% endif %}
```

NB: due to unavailable state , the jinja statement 'is defined' does not work (indeed the variable exists)

Reply:
Why are you transforming `unavailable` into `unknown` in this template? I'm not following that.

Anyways, it can also be written as `{% if states('sensor.train_departure_time') in ("unavailable", "unknown") %}`

That said, `unknown` should not be needed to return at all (as that is a so called `None` value), so the whole `else` part can be dropped too.

Reply:
I upate